serial-vault-services-snap/bin/serviceinit.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# Initialize and start the database service
+echo "Initialize the database"
+$SNAP/usr/bin/wrapper-initialize initdb
+$SNAP/usr/bin/wrapper-pg_ctl -D $SNAP_USER_COMMON/data -l $SNAP_USER_COMMON/logs/logfile start
```

Comment:
maybe it would be good to verify if this operation results in an error due to locale or even set directly LC_ALL=en_US.UTF-8